.tekton/collector-build.yaml

```diff
@@ -9,13 +9,13 @@ metadata:
     build.appstudio.redhat.com/target_branch: '{{target_branch}}'
     pipelinesascode.tekton.dev/max-keep-runs: "500"
     # TODO(ROX-21073): re-enable for all PR branches
-    pipelinesascode.tekton.dev/on-cel-expression: event == "pull_request" && (source_branch.contains("rhtap") || source_branch.contains("konflux"))
+    pipelinesascode.tekton.dev/on-cel-expression: (event == "push" && target_branch == "master") || (event == "pull_request" && (source_branch.contains("rhtap") || source_branch.contains("konflux")))
```

Comment:
Asking out of curiosity, since we are limiting the build to master and PRs with rhtap/konflux in the branch, is there some way we could wait for the build to finish from GHA? Has anyone come up with a solution for it? If so, I assume we could start integrating these builds into our regular pipeline and launching tests with these images.

Reply:
Konflux is not intended to replace GHA as the daily driver in the current phase. It seems too unreliable (pipeline timeouts, not starting builds because of too many concurrent builds, ...)

That said, there is [this action](https://github.com/stackrox/actions/blob/main/release/wait-for-image/README.md) that waits for an image to appear in Quay.io 😉

Reply:
> If so, I assume we could start integrating these builds into our regular pipeline and launching tests with these images.

Very much yes please! That'll help make sure Konflux not only builds but its results are functional.

> is there some way we could wait for the build to finish from GHA?

No, I don't know about existing solutions and would need to research to answer that. There's a `#konflux-users` channel where you could ask.
Also, a simple thing would be to poll repos until images become available, we know image tags upfront. I think, CI in stackrox/stackrox does that.

Reply:
Sounds good, I might give it a try when I get a minute. Thanks to you both for the answers!

Reply:
JFYI, we had https://issues.redhat.com/browse/ROX-22242 but I created https://issues.redhat.com/browse/ROX-24427 for explicitness, clarity and for us not to forget. Feel free to reuse this ROX-24427 or not.